qiskit/providers/aer/backends/aer_simulator.py

```diff
@@ -523,6 +523,7 @@ def _default_options(cls):
             matrix_product_state_truncation_threshold=1e-16,
             matrix_product_state_max_bond_dimension=None,
             mps_sample_measure_algorithm='mps_heuristic',
+            MPS_log_data=None,
```

Comment:
The doc string of the class and QasmSimulator should be updated to add a descirption of this option in the list of of options.  Looking here its not clear to me if this is a boolean value or not. If bool the default should probably be `False` instead of `None`.

Reply:
Changed to `False` and add description in `aer_simulator.py` and in `qasm_simulator.py`.